doc/HOWTO_RELEASE.rst.txt

```diff
@@ -19,18 +19,20 @@ Source tree
 NumPy Docs
 ----------
 - https://github.com/numpy/numpy/blob/master/doc/HOWTO_RELEASE.rst.txt
-- http://projects.scipy.org/numpy/wiki/MicrosoftToolchainSupport (dead link)
+
+.. - http://projects.scipy.org/numpy/wiki/MicrosoftToolchainSupport (dead link)
```

Comment:
Should "dead link" still be there?

Reply:
Sorry, @charris, I meant that to be commented out as well.  

What I *wanted* to do was leave both the link and the annotation completely alone. If I knew a way of "escaping" a URL so the tools would treat it like a plain string, that's what I would have done.

Reply:
I hadn't seen that before, does `..` actually work to make it a comment?

Reply:
We could provide a historical link for now: https://web.archive.org/web/20130402024847/http://projects.scipy.org/numpy/wiki/MicrosoftToolchainSupport (not sure if the 2012 version is not too old).

Reply:
Using `\http://projects.scipy.org/numpy/wiki/MicrosoftToolchainSupport` seems to make the link non-clickable for me, but an archive link is probably more useful anyway.

Reply:
Do these links have current relevance? I didn't look in archive.org because I was hesitant to resurrect links for a process that no longer exists.

Reply:
> does .. actually work to make it a comment?

I learned about it from 
https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#comments

Reply:
Oh... I suppose just commenting it out or even deleting may be fine as well here. I guess @charris was only wondering if it worked to hide/remove the link, which the `.. ` directive does...

Reply:
> wondering if it worked to hide/remove the link
Oh, I see. Looking at the docs it really could go either way; sorry @charris to misunderstand you.

> deleting may be fine
They were historic. Now they're history.